### ISSUE TITLE
evt.detail.intersection.point just shows the mousentered position 

### ISSUE THREAD

**Issue by sebi-b** [aframevr/aframe#2257]
Hey,

If I click on an intersected component, which fires a click event, the intersection point never shows the clicked point but just the position, where the cursor entered the component.

Any help?

my javascript:
```

var paper = document.querySelector("#paper");
paper.addEventListener('click', function(evt){
 console.log('clicked @: ', evt.detail.intersection.point);
});
```

my entity:

```
<a-entity geometry="primitive: plane; height: 6.1; width: 8" material="src: #p-dr-mtl" class="paper" id="p-dr" position="0 10 0" rotation="-35 0 0" scale="0 0 0">
            <a-animation attribute="position" begin="getPaper" to="0 5.25 -1.2"></a-animation>
            <a-animation attribute="scale" dur="2000" begin="getPaper" to="0.2 0.2 0.2"></a-animation>
</a-entity>
```

**Comment by ngokevin** [comment]
That's expected. 3D raycaster returns 3D point. If you want mouse click event, you can just listen to click on the window.

**Comment by donmccurdy** [comment]
@ngokevin I think the question/problem was that the intersection data was cached from the first intersection event, and not updated on each frame after that. We fixed that a while back though (by v0.3.2 at least) and I can't reproduce the problem now in v0.3.2 or v0.4.0.

Here's another script I'm using to test it:

```html
<script src="https://aframe.io/releases/0.4.0/aframe.min.js"></script>
<a-scene>
  <a-box position="0 0 -5"  class="intersectable" material="color: red"></a-box>
          <a-camera>
          <a-entity raycaster="far: 30; objects: .intersectable"
                    cursor
                    geometry="primitive: ring; radiusOuter: 0.015;
                              radiusInner: 0.01; segmentsTheta: 32"
                    material="color: #283644; shader: flat"
                    position="0 0 -0.75"></a-entity>
        </a-camera>
  </a-scene>
<script>
var box = document.querySelector('a-box');
box.addEventListener('click', (e) => console.log(e.detail.intersection.point))
</script>
```

^Note that the point reported is different depending on where we click the box (as expected).

**Comment by sebi-b** [comment]
Hey! Thank you. I had a wrong/old version of a-frame.
